beta/src/pages/learn/index.md

```diff
@@ -389,27 +388,33 @@ Hooks are more restrictive than regular functions. You can only call Hooks *at t
 
 ## Sharing data between components {/*sharing-data-between-components*/}
 
-In the previous example, each button had its own independent counter:
+In the previous example, each `MyButton` had its own independent `count`, and when each button was clicked, only the `count` for the button clicked changed:
 
-```js {2-4}
-- MyApp
-  - MyButton (count: 3)
-  - MyButton (count: 1)
-  - MyButton (count: 2)
-```
+<DiagramGroup>
+    <Diagram name="sharing_data_child.svg" alt="TODO: Sharing data" height={734} width={814}>
+        Before clicking, each MyButton has a count value set to zero.
+    </Diagram>
+    <Diagram name="sharing_data_child_clicked.svg" alt="TODO: Sharing data" height={734} width={814}>
+        After clicking, only one MyButton count value has updated.
+    </Diagram>
+</DiagramGroup>
 
-However, you'll often need components to *share data and always update together*.
+However, often you'll need components to *share data and always update together*.
 
-To make all buttons display the same `count` and update together, you need to move the state from the individual buttons "upwards" to the closest component containing all of them. In this example, it is `MyApp`:
+To make both `MyButton` components display the same `count` and update together, you need to move the state from the individual buttons "upwards" to the closest component containing all of them.
 
-```js {1}
-- MyApp (count: 3)
-  - MyButton
-  - MyButton
-  - MyButton
-```
+In this example, it is `MyApp`:
+
+<DiagramGroup>
+    <Diagram name="sharing_data_parent.svg" alt="TODO: Sharing data" height={770} width={820}>
+        Before clicking, count is stored in MyApp and passed down to both children as props.
+    </Diagram>
+    <Diagram name="sharing_data_parent_clicked.svg" alt="TODO: Sharing data" height={770} width={820}>
+        After clicking, count updates in MyApp and the update is passed to both children as props.
```

Comment:
"the update is passed" is a bit confusing. What's being passed is the new value of `count`. Can we slightly reword this to avoid the implication that an update can be passed?

Reply:
Yeah, I was trying to avoid "state" since that seems to be avoided in this section. I'll clarify!